canary_tests.sh

```diff
@@ -11,31 +12,17 @@ fi
 # See https://stackoverflow.com/a/1482133 for an explanation of this next line. It gets the
 # directory in which this script resides.
 this_dir="$(dirname -- "$( readlink -f -- "$0"; )")";
-echo "this_dir is $this_dir"
 pushd "$this_dir" > /dev/null
 
 # This script is adapted from
 # https://viam.atlassian.net/wiki/spaces/ENG/pages/155451406/Data+ML+Canary+Testing+Bot
 
-now=$(date +"%T")
-echo "Current time : $now"
-# stop the viam-server first
-echo "stopping viam-server"
 systemctl stop viam-canary
 
 # UPDATE RDK
 viam-server --aix-update
-if [ $? -eq 0 ]
-then
-  echo "rdk successfully updated RDK"
```

Comment:
Again making the justification for this more obvious: the `set -e` at the top of the file exits early if anything fails. This if statement will never fire because the only way to get this far into the script is for the server update to succeed.